digiwf-libs/digiwf-testing/digiwf-e2e-test-core/src/main/java/de/muenchen/oss/digiwf/integration/e2e/test/DigiwfE2eTest.java

```diff
@@ -15,6 +15,6 @@
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @ActiveProfiles("itest")
 @DirtiesContext
-@EmbeddedKafka(partitions = 1, brokerProperties = {"listeners=PLAINTEXT://localhost:19999"})
+@EmbeddedKafka
```

Comment:
Kann das der Grund sein, warum es nicht mehr parallel läuft?

Reply:
Fehler bei Maven `1C`

![image](https://github.com/it-at-m/digiwf-core/assets/10849776/c2fe0a40-d50e-4d55-a78e-2f85b00a6627)

Reply:
Also es liegt definitiv an der Kafka-Config. Das scheint aber auch ein kleiner Verhau zu sein, wenn wir in den Integrations-Service-E2E-Tests Basic Kafka Configs überschreiben, die von DigiWF-Message kommen 😐

Reply:
Die Analyse frisst aber ganz schön Zeit, wegen den langen Build-Zeiten🙈

Können wir auch ausklammern, aber wenn wir alle nun so lange warten müssen wegen den sequentiellen Builds, sollten wir das vielleicht doch nochmal anschauen

Reply:
> Das scheint aber auch ein kleiner Verhau zu sein

Ja hab ich mir auch schon gedacht, wollte jetzt aber in dem Ticket nicht noch anfangen den Message-Starter anzufassen.

> warum es nicht mehr parallel läuft?

Nachdem das auf dem dev ja auch der Fall ist, würde ich das hier ausklammern oder was meinst du?

Reply:
Ja passt